opennms-doc/guide-admin/src/asciidoc/text/provisioning/getting-started.adoc

```diff
@@ -90,6 +90,30 @@ We will then give an example of how to be more _directed_ and deliberate about y
 Automatically discovered entities are analyzed, persisted to the relational data store, and then managed based on the policies defined in the default foreign source definition.
 This is very similar to the way that entities were previously handled by the (now obsolete) Capsd daemon but with finer grained sense of control.
 
+==== Encrypting SNMP Credentials
+
+You can save SNMP credentials like readCommunity/WriteCommunity, authPassPhrase/privPassPhrase on the file system in encrypted format.
+
+===== Encryption methodology
+
+A random password is generated (once) for `snmp-config` alias and it is stored in java keystore file at `$OPENNMS_HOME/etc/scv.jce`.
+This password is used to encrypt/decrypt SNMP credentials.
```

Comment:
```suggestion
This password encrypts/decrypts SNMP credentials.
```
Does the password do this, or does one require the password so SNMP credentials can be encrypted/decrypted?